Healthy/Classes/UseCases/RegisterUseCase.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import Factory
+import Domain
+import Networking
+
+extension Container {
```

Comment:
Let's add it to another file in the same folder

Reply:
can you more explain about this comment ? @ahmdmhasn